components/raftstore-v2/src/operation/ready/apply_trace.rs

```diff
@@ -519,6 +522,17 @@ impl<EK: KvEngine, ER: RaftEngine> Storage<EK, ER> {
         }
         let region_id = self.region().get_id();
         let raft_engine = self.entry_storage().raft_engine();
+        let epoch = raft_engine
+            .get_region_state(region_id, trace.admin.flushed)
+            .unwrap()
+            .unwrap()
+            .get_region()
+            .get_region_epoch()
+            .clone();
+        if util::is_epoch_stale(self.flushed_epoch(), &epoch) {
+            write_task.flushed_epoch = Some(epoch);
+        }
```

Comment:
It may cause some IO, Maybe we could compare `self.region().get_region_epoch()`?

```suggestion
        if util::is_epoch_stale(self.flushed_epoch(), self.region().get_region_epoch()) {
            write_task.flushed_epoch = Some(self.region().get_region_epoch().clone());
        }
```

Reply:
all the region states are in memory.